frinx-uniconfig/developer-guide/metrics/readme.md

```diff
@@ -55,7 +57,7 @@ Metrics.globalRegistry.gauge(UniconfigTransactionManager.class.getName() + ".ope
 
 ## Tags 
 
-Tags are currently not available in the version *4.2.x*, although support for them is planned for future major release.
+Tags are currently not available in the version *7.0.x*, although support for them is planned for future major release.
```

Comment:
You can remove the whole tag section, this was related to the version of dropwizard metrics.